payment-library-webview-module/src/main/java/com/payu/android/front/sdk/payment_library_webview_module/web/authorization/client/WebAuthorizationViewChromeClient.java

```diff
@@ -27,6 +45,51 @@ public void onProgressChanged(WebView view, int newProgress) {
         }
     }
 
+    @Override
+    public void onPermissionRequest(PermissionRequest request) {
+        String[] requestedResources = request.getResources();
+        List<String> requiredPermissions = new ArrayList<>();
```

Comment:
Strasznie dużo wydzielonych zmiennych lokalnych, które nie są potrzebne i przez to komplikują kod 
```
        if (cameraNeeded) {
            pendingRequest = request;
            ActivityCompat.requestPermissions(
                    webPaymentActivity,
                    new String[] { Manifest.permission.CAMERA },
                    CAMERA_PERMISSION_REQUEST_CODE
            );
        } else {
            request.grant(requestedResources);
        }
```